fem/doftrans.hpp

```diff
@@ -80,6 +80,9 @@ class StatelessDofTransformation
    inline int Width() const { return size_; }
    inline int NumCols() const { return size_; }
 
+   /// If the DofTransformation performs no transformation
+   virtual bool IsEmpty() const = 0;
```

Comment:
Is this the right name for this?  IsTrivial()?  IsIdentity()?

Reply:
Excellent go ahead and get that in and I'll approve.

Reply:
[aa8e466](https://github.com/mfem/mfem/pull/3820/commits/aa8e466e42b36fcb93f2ff9df25345351c30e3f0)